File: problems/001-anagram-detection/rust/practice.rs
use crate::json::JVal;

pub fn solve(c: &JVal) -> JVal {
    let s = c.get("s").as_str();
    let t = c.get("t").as_str();
    // Return true if s and t are anagrams of each other, false otherwise.
    JVal::Bool(false)
}

File: tools/zig-behave-spike/b01/practice.zig
const std = @import("std");
const json = std.json;

pub fn solve(alloc: std.mem.Allocator, c: json.Value) !json.Value {
    _ = alloc;
    const s = c.object.get("s").?.string;
    const t = c.object.get("t").?.string;
    // Return true if s and t are anagrams of each other, false otherwise.
    _ = s;
    _ = t;
    return .{ .bool = false };
}
